### ISSUE TITLE
Error saving transactions

### ISSUE THREAD

**Issue by xzaz** [JC5/firefly-iii#136]
With the new version 3.5.6 I get a exception when saving a transaction:

Next exception 'Illuminate\Database\QueryException' with message 'SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'transaction_journal_id' cannot be null (SQL: inse$
Stack trace:

Do you need the whole stacktrace?

**Comment by JC5** [comment]
Yes please, I'm curious what's wrong. The log line should continue after the $, include that one too. 

- What kind of transaction was it?
- Did you create a _new_ category when doing so?
- Did you refer to a piggy bank?
- Did you use tags?

**Comment by xzaz** [comment]
Full stacktrace (anonymized) 

```
Next exception 'Illuminate\Database\QueryException' with message 'SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'transaction_journal_id' cannot be null (SQL: insert into `category_transaction_journal` (`category_id`, `transaction_journal_id`) values (31, ))' in /my-url/htdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Connection.php:631
Stack trace:
#0 /my-url/htdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Connection.php(591): Illuminate\Database\Connection->runQueryCallback('insert into `ca...', Array, Object(Closure))
#1 /my-url/htdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Connection.php(381): Illuminate\Database\Connection->run('insert into `ca...', Array, Object(Closure))
#2 /my-url/htdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Connection.php(337): Illuminate\Database\Connection->statement('insert into `ca...', Array)
#3 /my-url/htdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1719): Illuminate\Database\Connection->insert('insert into `ca...', Array)
#4 /my-url/htdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Relations/BelongsToMany.php(868): Illuminate\Database\Query\Builder->insert(Array)
#5 /my-url/htdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Relations/BelongsToMany.php(554): Illuminate\Database\Eloquent\Relations\BelongsToMany->attach('31', Array, true)
#6 /my-url/htdocs/firefly-iii/app/Repositories/Journal/JournalRepository.php(189): Illuminate\Database\Eloquent\Relations\BelongsToMany->save(Object(FireflyIII\Models\Category))
#7 /my-url/htdocs/firefly-iii/app/Http/Controllers/TransactionController.php(309): FireflyIII\Repositories\Journal\JournalRepository->store(Array)
#8 [internal function]: FireflyIII\Http\Controllers\TransactionController->store(Object(FireflyIII\Http\Requests\JournalFormRequest), Object(FireflyIII\Repositories\Journal\JournalRepository), Object(FireflyIII\Helpers\Attachments\AttachmentHelper), 'transfer')
#9 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(8424): call_user_func_array(Array, Array)
#10 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(8493): Illuminate\Routing\Controller->callAction('store', Array)
#11 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(8473): Illuminate\Routing\ControllerDispatcher->call(Object(FireflyIII\Http\Controllers\TransactionController), Object(Illuminate\Routing\Route), 'store')
#12 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#13 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(9130): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#15 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(9112): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#16 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(8474): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#17 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(8459): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(FireflyIII\Http\Controllers\TransactionController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'store')
#18 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(7434): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'FireflyIII\\Http...', 'store')
#19 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(7405): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#20 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(7058): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#21 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#22 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(9130): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#23 /my-url/htdocs/firefly-iii/app/Http/Middleware/Range.php(83): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 [internal function]: FireflyIII\Http\Middleware\Range->handle(Object(Illuminate\Http\Request), Object(Closure))
#25 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#26 /my-url/htdocs/firefly-iii/app/Http/Middleware/Authenticate.php(74): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 [internal function]: FireflyIII\Http\Middleware\Authenticate->handle(Object(Illuminate\Http\Request), Object(Closure))
#28 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#29 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(9112): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#31 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(7059): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#32 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(7047): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#33 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(7032): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#34 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(2066): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#35 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#36 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(9130): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#37 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(2687): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 /my-url/htdocs/firefly-iii/app/Http/Middleware/VerifyCsrfToken.php(25): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#39 [internal function]: FireflyIII\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#40 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#41 /my-url/htdocs/firefly-iii/app/Http/Middleware/ReplaceTestVars.php(54): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#42 [internal function]: FireflyIII\Http\Middleware\ReplaceTestVars->handle(Object(Illuminate\Http\Request), Object(Closure))
#43 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#44 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(12378): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#45 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#46 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#47 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(11027): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#48 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#49 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#50 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(12116): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#51 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#52 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#53 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(12055): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#54 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#55 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#56 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(2737): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#57 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#58 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#59 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#60 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(9112): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#61 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(2013): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#62 /my-url/htdocs/firefly-iii/bootstrap/cache/compiled.php(1999): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#63 /my-url/htdocs/firefly-iii/public/index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#64 {main}  
```


    What kind of transaction was it? Transfer
    Did you create a new category when doing so? No
    Did you refer to a piggy bank? No
    Did you use tags? No

**Comment by JC5** [comment]
Will get right to it!

**Comment by xzaz** [comment]
Thanks,

There is more:

```
[2015-12-28 14:53:11] production.ERROR: exception 'PDOException' with message 'SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'transaction_journal_id' cannot be null' in /my-urlhtdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Connection.php:380
Stack trace:
#0 /my-urlhtdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Connection.php(380): PDOStatement->execute(Array)
#1 /my-urlhtdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Connection.php(624): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'insert into `ca...', Array)
#2 /my-urlhtdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Connection.php(591): Illuminate\Database\Connection->runQueryCallback('insert into `ca...', Array, Object(Closure))
#3 /my-urlhtdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Connection.php(381): Illuminate\Database\Connection->run('insert into `ca...', Array, Object(Closure))
#4 /my-urlhtdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Connection.php(337): Illuminate\Database\Connection->statement('insert into `ca...', Array)
#5 /my-urlhtdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1719): Illuminate\Database\Connection->insert('insert into `ca...', Array)
#6 /my-urlhtdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Relations/BelongsToMany.php(868): Illuminate\Database\Query\Builder->insert(Array)
#7 /my-urlhtdocs/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Relations/BelongsToMany.php(554): Illuminate\Database\Eloquent\Relations\BelongsToMany->attach('31', Array, true)
#8 /my-urlhtdocs/firefly-iii/app/Repositories/Journal/JournalRepository.php(189): Illuminate\Database\Eloquent\Relations\BelongsToMany->save(Object(FireflyIII\Models\Category))
#9 /my-urlhtdocs/firefly-iii/app/Http/Controllers/TransactionController.php(309): FireflyIII\Repositories\Journal\JournalRepository->store(Array)
#10 [internal function]: FireflyIII\Http\Controllers\TransactionController->store(Object(FireflyIII\Http\Requests\JournalFormRequest), Object(FireflyIII\Repositories\Journal\JournalRepository), Object(FireflyIII\Helpers\Attachments\AttachmentHelper), 'transfer')
#11 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(8424): call_user_func_array(Array, Array)
#12 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(8493): Illuminate\Routing\Controller->callAction('store', Array)
#13 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(8473): Illuminate\Routing\ControllerDispatcher->call(Object(FireflyIII\Http\Controllers\TransactionController), Object(Illuminate\Routing\Route), 'store')
#14 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#15 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(9130): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#16 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#17 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(9112): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#18 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(8474): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#19 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(8459): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(FireflyIII\Http\Controllers\TransactionController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'store')
#20 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(7434): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'FireflyIII\\Http...', 'store')
#21 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(7405): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#22 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(7058): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#24 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(9130): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#25 /my-urlhtdocs/firefly-iii/app/Http/Middleware/Range.php(83): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#26 [internal function]: FireflyIII\Http\Middleware\Range->handle(Object(Illuminate\Http\Request), Object(Closure))
#27 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#28 /my-urlhtdocs/firefly-iii/app/Http/Middleware/Authenticate.php(74): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 [internal function]: FireflyIII\Http\Middleware\Authenticate->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#31 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(9112): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#33 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(7059): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#34 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(7047): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#35 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(7032): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#36 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(2066): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#37 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#38 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(9130): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#39 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(2687): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#40 /my-urlhtdocs/firefly-iii/app/Http/Middleware/VerifyCsrfToken.php(25): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#41 [internal function]: FireflyIII\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#42 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#43 /my-urlhtdocs/firefly-iii/app/Http/Middleware/ReplaceTestVars.php(54): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 [internal function]: FireflyIII\Http\Middleware\ReplaceTestVars->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#46 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(12378): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#47 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#48 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#49 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(11027): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#51 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#52 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(12116): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#53 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#54 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#55 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(12055): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#56 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#57 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#58 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(2737): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#59 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#60 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(9122): call_user_func_array(Array, Array)
#61 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#62 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(9112): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#63 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(2013): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#64 /my-urlhtdocs/firefly-iii/bootstrap/cache/compiled.php(1999): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#65 /my-urlhtdocs/firefly-iii/public/index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#66 {main}
```

**Comment by JC5** [comment]
Should be fixed with the immediate availability of 3.5.6.1.
